addons/mail/static/src/js/composer.js

```diff
@@ -479,16 +479,21 @@ var BasicComposer = Widget.extend(chat_mixin, {
     },
 
     send_message: function () {
+        var button = $(".o_composer_button_send");
```

Comment:
`this` contains the composer only. You can probably find a better selector instead of looking on the whole DOM